database/fake_data.sql

```diff
@@ -103,7 +103,7 @@ VALUES
 (89, 4),
 (19, 10),
 (76, 2),
-(41, 13)
+(41, 13),
```

Comment:
I didn't spot this very good catch

Reply:
LGTM. These kind of things slip through unless we run it on an example database, good update.

Reply:
Great learning opportunity for the deputy quality assurer